DBSCE/LocalKeyHelper-Mac.md

```diff
@@ -0,0 +1,96 @@
+<span style="color:red">!!! DRAFT - Please note this page is still in draft as we work on platform specific details
+</span>
+
+### Local key helper on Mac
+
+On macOS, a Local Key Helper is a protocol (`LocalKeyHelper.h`), and its implementation (`LocalKeyHelperImpl.h`/`LocalKeyHelperImpl.m`) is yet to be determined.
+
+The Local Key Helper will be implemented and shipped as an XPC service on macOS. It will be signed with app sandbox entitlement for securety requirement.
```

Comment:
Suggest rewording:

Each local key helper vendor will implement and ship an XPC service on macOS.